.github/actions/cache/action.yml

```
@@ -33,16 +33,16 @@ runs:
     shell: bash
 
   - name: Determine caches
-    env:
-      PWD: $(pwd)
+    id: cache_dirs
     run: |
-      python -c "from pip._internal.locations import USER_CACHE_DIR; print('PIP_CACHE_DIR=' + USER_CACHE_DIR)" >> $GITHUB_ENV
+      pip_dir=$(python -c "from pip._internal.locations import USER_CACHE_DIR; print(USER_CACHE_DIR)")
+      echo "pip_dir=$pip_dir" >> $GITHUB_OUTPUT
```

nit: `pip cache dir` should be supported in all operating systems
```suggestion
      echo "pip_dir=$(pip cache dir)" >> $GITHUB_OUTPUT
```